README.md

```diff
@@ -298,6 +354,95 @@ by its ID with the `custom.` prefix:
 />
 ```
 
+### Custom Queries within the catalog-info.yaml file of the Backstage Entity
+
+You can also register your custom queries in the catalog-info.yaml file.
+
+```yaml
+apiVersion: backstage.io/v1alpha1
+kind: Component
+metadata:
+  name: demo-backstage
+  description: Backstage Demo instance.
+  annotations:
+    backstage.io/kubernetes-id: kubernetescustom
+spec:
+  type: website
+  owner: user:default/mjakl
+  lifecycle: experimental
+  system: integrations
+  queries:
+    - id: Problem Events
+      description: Fetch Problem Events
+      query: >
+        fetch events, from: -2d
+              | filter event.kind=="DAVIS_PROBLEM"
+              | fieldsAdd category=if(isNull(event.category), "N/A", else:
+        event.category)
+              | fieldsAdd id=if(isNull(event.id), "N/A", else: event.id)
+              | fieldsAdd status=if(isNull(event.status), "N/A", else:
+        event.status)
+              | fieldsAdd name=if(isNull(event.name), "N/A", else: event.name)
+              | fieldsAdd environment = "${environmentName}"
```

Comment:
@blasget should it be a `$${environmentName}`

Reply:
@johannes-b from the README.md: 

As mentioned before, queries can contain placeholders. In the catalog-info.yaml
file, the placeholders are prefixed with a **single `$`**.

Reply:
Why is it different to the `app-config.yaml` where I would use $$?

Reply:
The plugin utilizes its own set of environment variables, which require a single `$` for substitution. Variables defined with a single `$` in the `app-config.local` file are treated as environment variables by Backstage itself. If any environment variables are missing, the entire configuration value will evaluate to undefined.

The environment variables from the Dynatrace plugin can change depending on the context (e.g., the environmentName can vary with each query execution when there are multiple environments defined). To handle its own environment variables, Dynatrace uses the substitution syntax `$${...}`, which resolves to `${...}`.

More information here: https://backstage.io/docs/conf/writing/#environment-variable-substitution

Can you confirm this @Kirdock ?

Reply:
Yes, correct. The app.config.yaml supports env variables and our ones should be escaped and the catalog-info does not support env variables

Reply:
thanks for the explanation!